Pool/HighScores.vb

```diff
@@ -3,17 +3,17 @@ Imports System.Linq
 Imports Pool.Logging
 Public Class HighScores
 #Region "Global to form"
-    Dim highScoreTheme As New ScoreTheme(Me)
     Dim player As New PlayerStats
     Dim games As New Games
-    'Dim screen As New AppState
     Dim allPlayers As New Hashtable
     Dim allGames As New Hashtable
     Dim highScores As New DataSet
     Dim addLoc As New Point
     Dim saveLoc As New Point
+    Dim pvpLoc As New Point
     Dim saveDim As New Drawing.Size
     Dim addDim As New Drawing.Size
+    Dim pvpDim As New Drawing.Size
```

Comment:
See if these can be taken out if we are not using them, we reworked controls to fix these won't be needed.

Reply:
These were not being used anymore. They were removed along with its references.